src/map/pc.cpp

```diff
@@ -6260,12 +6260,13 @@ bool pc_isUseitem(map_session_data *sd,int n)
 		return false; // Mercenary Scrolls
 
 	if( item->flag.group || item->type == IT_CASH) {	//safe check type cash disappear when overweight [Napster]
-		if( pc_is90overweight(sd) ) {
-			clif_msg(sd, ITEM_CANT_OBTAIN_WEIGHT);
+		if (pc_is90overweight(sd)) {
+			clif_msg_color(sd, ITEM_CANT_OBTAIN_WEIGHT, color_table[COLOR_RED]);
 			return false;
 		}
-		if( !pc_inventoryblank(sd) ) {
-			clif_messagecolor(&sd->bl, color_table[COLOR_RED], msg_txt(sd, 732), false, SELF); //Item cannot be open when inventory is full
+		// On official servers you can't open a box item if you don't have at least 11 spaces in your inventory
+		if (pc_inventoryblank(sd) < 11) {
```

Comment:
battle_conf?

Reply:
I've changed it to <= 10, but ended up leaving the value hardcoded with a TODO comment, as this is probably calculated from the total items the box can give.
I'll test some more and fix it in a future PR.

Reply:
I confirmed the behavior and fixed it in this PR